liquibase-cli/src/main/java/liquibase/integration/commandline/LiquibaseCommandLine.java

```diff
@@ -104,7 +104,7 @@ private void cleanup() {
 
     public LiquibaseCommandLine() {
         this.legacyPositionalArguments = new HashMap<>();
-        this.legacyPositionalArguments.put("calculatechecksum", CalculateChecksumCommandStep.CHANGESET_IDENTIFIER_ARG.getName());
```

Comment:
I think we should return this one to its previous state as well.

Reply:
Reverted to changeset identifier